pettingzoo/classic/connect_four/connect_four.py

```diff
@@ -285,6 +285,8 @@ def render(self):
         if self.render_mode == "human":
             pygame.display.update()
             self.clock.tick(self.metadata["render_fps"])
+            pygame.display.flip()
```

Comment:
You should only need one of flip and update, and you should check docs and stuff to see but I imagine pomp might be called before these others.

Reply:
I can't find much on the order of `pump` and `update`, but you are correct with not needing both `update` and `flip`